src/app/api/login/route.ts

```diff
@@ -0,0 +1,22 @@
+//import admin from "firebase-admin";
+import { projectAuth } from "@/firebase/config";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(request: NextRequest){
+    try{
+        const {email, password} = await request.json();
+
+        //sign in user
+        const user = await projectAuth.signInWithEmailAndPassword(email, password);
+
+        //get the token from user
```

Comment:
Check if user DNE for whatever reason, log that, return an error (can't use the service without an idToken)